statistics/handle/update.go

```diff
@@ -436,6 +436,7 @@ func (h *Handle) UpdateStatsByLocalFeedback(is infoschema.InfoSchema) {
 			newIdx.CMSketch = statistics.UpdateCMSketch(idx.CMSketch, eqFB)
 			newIdx.Histogram = *statistics.UpdateHistogram(&idx.Histogram, &statistics.QueryFeedback{Feedback: ranFB})
 			newIdx.Histogram.PreCalculateScalar()
+			newIdx.Flag = 0
```

Comment:
Only unset `AnalyzeFlag` bit in case we use other bits of this flag in the future?